functions/.runtimeconfig.json.example

```diff
@@ -14,5 +14,8 @@
   "agora": {
     "app_id": "",
     "app_certificate": ""
+  },
+  "flag": {
+    "autoadmin": ""
   }
```

Comment:
An example of default/empty `flag` object that can be used to switch the _Auto Admin On Register_ functionality per Firestore environment. Any truthy value should suffice.